account-kit/signer/src/client/index.ts

```diff
@@ -645,6 +658,112 @@ export class AlchemySignerWebClient extends BaseSignerClient<ExportWalletParams>
     const nonce = this.getOauthNonce(publicKey);
     return this.request("/v1/prepare-oauth", { nonce });
   };
+
+  /**
+   * Retrieves the list of MFA factors configured for the current user.
+   *
+   * @returns {Promise<{ factors: MfaFactor[] }>} A promise that resolves to an array of configured MFA factors
+   * @throws {NotAuthenticatedError} If no user is authenticated
+   */
+  public override getMfaFactors = async (): Promise<{
+    multiFactors: MfaFactor[];
+  }> => {
+    if (!this.user) {
+      throw new NotAuthenticatedError();
+    }
+
+    const stampedRequest = await this.turnkeyClient.stampGetWhoami({
+      organizationId: this.user.orgId,
+    });
+
+    return this.request("/v1/auth-list-multi-factors", {
```

Comment:
Worth gut checking to see if this request already includes the jwt in the headers? https://github.com/alchemyplatform/aa-sdk/blob/008812b0/account-kit/signer/src/client/base.ts#L374 We might not even need to have the stamped request object in there.